library/std/src/f128.rs

```diff
@@ -235,10 +237,14 @@ impl f128 {
 
     /// Calculates Euclidean division, the matching method for `rem_euclid`.
     ///
-    /// This computes the integer `n` such that
+    /// In infinite precision this computes the integer `n` such that
```

Comment:
Usually I see this arranged as "This computes the operation on the integer `n` with infinite precision"

Reply:
It's a nit, but I'd say the calculation is done "in" infinite precision rather than "with" it.

Reply:
Those are fine too. It's honestly at least partly that "In infinite precision" feels like an odd way to start the sentence.